kernel/selinux/sepolicy.c

```diff
@@ -679,6 +688,56 @@ static bool add_type(struct policydb *db, const char *type_name, bool attr)
 				0);
 	}
 
+	return true;
+#elif CONFIG_IS_HW_HISI == 1
```

Comment:
Is `== 1` needed?

Reply:
I want to say yes,but I will try simpler code to implement functionality.

Reply:
> Is `== 1` needed?

Huawei HISI SELinux Code is come from 5.10.x.I compared the SELinux code of the original Linux 5.10. x with Huawei's 4.9 SELinux code, and basically achieved a similarity.So I want to indicate a marker to indicate that it needs to call the following code. VScode seems to always ignore the part below the mark that the code should run. However, during the compilation process, it is possible.

Reply:
> Is `== 1` needed?

HISI_ SELINUX_ EBITMAP_ RO function must be disabled in the Huawei kernel, otherwise it cannot be compiled. But it still needs to be used in the code, otherwise it will prompt for missing parameters. I'll take this HISI_ SELINUX_ EBITMAP_ RO  use as a marker to determine if it is the Huawei 4.9 kernel.I don't really have a better idea.

Reply:
Right Now I revised the code to simplify it.

Reply:
> Is `== 1` needed?

This code no longer exists